src/components/Card.svelte

```diff
@@ -6,6 +6,7 @@
   import { ProjectThemes, ProjectThemesFa } from "../enums/themes.enum";
   import "../style/typography.css";
   import "../style/card-theme.css";
+  import { ButtonDisplayOption } from "../enums/button-selector.enum";
 
```

Comment:
```suggestion
export let buttonLabel: string;
```

do this then buttonLabel prop will be able to be passed from entries